packages/app-builder-lib/templates/nsis/include/allowOnlyOneInstallerInstance.nsh

```diff
@@ -37,14 +33,9 @@
     !endif
 !macroend
 
-!macro FIND_PROCESS _FILE _ERR
-  !ifdef INSTALL_MODE_PER_ALL_USERS
-    ${nsProcess::FindProcess} "${_FILE}" ${_ERR}
-  !else
-    # find process owned by current user
-    nsExec::Exec `"$SYSDIR\cmd.exe" /c tasklist /FI "USERNAME eq %USERNAME%" /FI "IMAGENAME eq ${_FILE}" /FO csv | "$SYSDIR\find.exe" "${_FILE}"`
-    Pop ${_ERR}
-  !endif
+!macro FIND_PROCESS _PATH _ERR
+  nsExec::Exec `powershell -Command "if ((Get-Process | Where-Object {$$_.Path -and $$_.Path.StartsWith('${_PATH}')}).Count -gt 0) {exit 0} else {exit 1}"`
```

Comment:
Using PowerShell is not highly recommended, as many enterprise and institutional users have PowerShell disabled for security reasons, which would cause these scripts to fail.

Reply:
I suggest making this feature configurable, allowing users to choose whether to use PowerShell。

Reply:
The "user" as in the developer using electron-builder, or the end user?

If the latter, we can't guarantee if powershell is accessible, or if they know whether it is accessible. We also need to take into account the update flow where it's non-interactive.

Not sure if possible, but is there a way to detect if powershell is available? If so, use the powershell command in this PR, if not, fallback to previous logic?

Reply:
I did commit with fallback, but need more tests.